.github/workflows/build-images.yaml

```diff
@@ -121,32 +121,32 @@ jobs:
       - id: set_is_parent_modified
         run: echo "::set-output name=is_parent_modified::${MODIFIED}"
     
-  hhvm:
-    runs-on: ubuntu-latest
-    needs: base-os
-    env:
-      DH_REPO: "cloudsuite/${{ github.job }}"
-    outputs:
-      is_parent_modified: ${{ steps.set_is_parent_modified.outputs.is_parent_modified }}
-    strategy:
-      matrix:
-        tag: ["3.30"]
-        platform: ["linux/amd64"]
-    steps:
-      - name: checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - if: ${{ needs.base-os.outputs.is_parent_modified == 'true' }}
-        run: echo "IS_PARENT_MODIFIED=true" >> $GITHUB_ENV
-      - name: build and push
-        run: "./.github/scripts/build-images.sh"
-        env:
-          IMG_TAG: "${{ matrix.tag }}"
-          DF_PATH: "./commons/${{ github.job }}/${{ matrix.tag }}"
-          DBX_PLATFORM: ${{ matrix.platform }}
-      - id: set_is_parent_modified
-        run: echo "::set-output name=is_parent_modified::${MODIFIED}"
+  # hhvm:
+  #   runs-on: ubuntu-latest
+  #   needs: base-os
+  #   env:
+  #     DH_REPO: "cloudsuite/${{ github.job }}"
+  #   outputs:
+  #     is_parent_modified: ${{ steps.set_is_parent_modified.outputs.is_parent_modified }}
+  #   strategy:
+  #     matrix:
+  #       tag: ["3.30"]
+  #       platform: ["linux/amd64"]
+  #   steps:
+  #     - name: checkout
+  #       uses: actions/checkout@v2
+  #       with:
+  #         fetch-depth: 0
+  #     - if: ${{ needs.base-os.outputs.is_parent_modified == 'true' }}
+  #       run: echo "IS_PARENT_MODIFIED=true" >> $GITHUB_ENV
+  #     - name: build and push
+  #       run: "./.github/scripts/build-images.sh"
+  #       env:
+  #         IMG_TAG: "${{ matrix.tag }}"
+  #         DF_PATH: "./commons/${{ github.job }}/${{ matrix.tag }}"
+  #         DBX_PLATFORM: ${{ matrix.platform }}
+  #     - id: set_is_parent_modified
+  #       run: echo "::set-output name=is_parent_modified::${MODIFIED}"
```

Comment:
If we deprecate the workload, do we need to keep these lines as comments in the script? It makes it ugly and does not serve any purpose if we won't build them again in the future.